CHANGELOG.md

```diff
@@ -120,6 +120,8 @@ New modules
 
 * `Data.Sign.Show` to show a sign
 
+* `Relation.Nullary.Irrelevant` defining the concept of irrelevance (h-proposition), where all elements of a type are equal.
```

Comment:
While this is a new module, it really is a refactoring, so this isn't quite the right place to put this. I'm sure previous CHANGELOGs have such entries that can be used as a guide on how to do this.

Reply:
@JacquesCarette I disagree with you here, but not enough to block this being merged as-is.